### PR TITLE
Readthedocs development

### PR DESCRIPTION
**Pull request by tmcclintock**
The python docstrings automatically generate the readthedocs documentation. Files for doing this are included in the `docs` directory. Additionally, updates and improvements to the docstrings for the `.py` files are included in this PR.

### CODE REVIEW

File: pyccl/massfunction.py [hunk 1]
```diff
@@ -25,45 +23,47 @@ def massfunc(cosmo, halo_mass, a, odelta=200):
...
-        sigmaM (float or array_like): RMS variance of halo mass.
+        float or array_like: RMS variance of halo mass.
 
     """
     return _vectorize_fn2(lib.sigmaM, 
                           lib.sigmaM_vec, cosmo, halo_mass, a)
 
 def halo_bias(cosmo, halo_mass, a, odelta=200):
-    """Halo bias.
+    """Tinker et al. (201) halo bias.
```
Comment (elisachisari) [inline pyccl/massfunction.py]: Small typo

File: pyccl/neutrinos.py [hunk 2]
```diff
@@ -36,16 +36,18 @@ def Omeganuh2(a, Neff, mnu, TCMB=2.725):
...
 def Omeganuh2_to_Mnu(a, Neff, OmNuh2, TCMB=2.725):
-    """Omeganu2h_to_Mnu
+    """Convert Omeganuh2 to neutrino mass Mnu.
+
+    Note: for all practical purposes, Neff is simply N_nu_mass.
 
     Args:
         a (float or array_like): Scale factor(s), normalized to 1 today.
-        Neff (float): Number of relativistic neutrino species (NB: to all practical purposes, Neff is simply N_nu_mass)
+        Neff (float): Number of relativistic neutrino species.
```
Comment (elisachisari) [inline pyccl/neutrinos.py]: Can we please keep the comment above for now?

File: pyccl/correlation.py [hunk 3]
```diff
@@ -17,18 +20,19 @@
...
 
     Args:
         cosmo (:obj:`Cosmology`): A Cosmology object.
-        ell (array_like): Multipoles corresponding to the input angular power spectrum
+        ell (array_like): Multipoles corresponding to the input angular power spectrum.
         C_ell (array_like): Input angular power spectrum.
         theta (float or array_like): Angular separation(s) at which to calculate the angular correlation function (in degrees).
         corr_type (string): Type of correlation function. Choices: 'GG' (galaxy-galaxy), 'GL' (galaxy-shear), 'L+' (shear-shear, xi+), 'L-' (shear-shear, xi-).
-        method (string, optional): Method to compute the correlation function. Choices: 'Bessel' (direct integration over Bessel function), 'FFTLog' (fast integration with FFTLog), 'Legendre' (brute-force sum over Legendre polynomials).
+        method (string, optional): Method to compute the correlation function.
```
Comment (elisachisari) [inline pyccl/correlation.py]: Can we keep the specification of the possible choices?

File: pyccl/neutrinos.py (hunk 2 above)
Comment (elisachisari) [reply pyccl/neutrinos.py]: Oh, never mind, you moved it above that line. That's ok.

**Comment by tmcclintock** [comment]
Yup, I'll fix these. Somehow they didn't get flagged when I did the merge.

**Comment by elisachisari** [comment]
@tmcclintock Could you add our citation clause from the README to this? Also, could you update the installation following the recent merge from @EiffL?

**Comment by tmcclintock** [comment]
The RTD pages now contain exactly the text in the README.md as of this post. @elisachisari the conflicts seem easy to resolve, but I don't know what the lines added to the `__init__.py` file are, so I'll let you do the merge.

**Comment by tmcclintock** [comment]
@EiffL thanks for that!

**Comment by tmcclintock** [comment]
Note: it appears that the automatically generated API on readthedocs has broken, even though it works fine locally. This PR shouldn't be accepted until it works again.

**Comment by damonge** [comment]
Checked and removed @elisachisari

**Pull request merged**